napari/layers/labels/labels.py

```diff
@@ -291,10 +291,10 @@ def __init__(
             contour=Event,
         )
 
-        if properties is not None:
-            self.properties = properties
-        else:
-            self.features = features
+        self._feature_manager = _FeatureManager.from_layer(
+            features=features, properties=properties
+        )
```

Comment:
My understanding is that 
1) feature will be ignored if properties is not None, 
should that be said in above docstring / and or emit a warning.
2) feature is the new way of doing things, so maybe it should be the one prioritized. 
3) the event above doe not emit feature but still does properties, I am completly wrong ?